test/cql-pytest/README.md

```diff
@@ -37,8 +37,16 @@ to control which tests to run:
   once, and also counts for you how many of the runs failed.
   For `pytest` to support the `--count` option, you need to install a
   pytest extension: `pip install pytest-repeat`
+* Additional useful pytest options, especially useful for debugging tests:
+  * -v: show the names of each individual test running instead of just dots.
+  * -s: show the full output of running tests (by default, pytest captures the test's output and only displays it if a test fails)
 
-Additional useful pytest options, especially useful for debugging tests:
+### How to run from PyCharm IDE
+* Create pytest config
+* Under `target` select `Module name`
+ * filename - to run all tests in the file (`test_keyspace`) 
+ * filename.testname - to run specific test (`test_keyspace.test_concurrent_create_and_drop_keyspace`) 
+* Add additional params under `Additional Arguments`
+
+![/images/pycharm_pytest_config](../images/pycharm_pytest_config.png)
```

Comment:
I don't think we should have a test/images directory, it's too strange... In any case, the look of pycharm will probably change so I'm not sure how this image will be helpful a year from now.

Reply:
Yes, it's strange, but the config should not change because it's python config and not relevant to any Pycharm version.

I remove it from the pr.